lib/resource/resource.ex

```diff
@@ -94,6 +94,13 @@ defmodule AshPaperTrail.Resource do
         doc: """
         Extensions that should be used by the version resource. For example: `extensions: [AshGraphql.Resource], notifier: [Ash.Notifiers.PubSub]`
         """
+      ],
+      store_inputs?: [
+        type: :boolean,
+        default: false,
+        doc: """
+        Whether or not to store the original inputs to the action as well as the changes within the version resource.
```

Comment:
```suggestion
        Whether or not to store the original inputs to the action as well as the changes within the version resource.  Note: There's no filtering of the user input, so if your resource has password attributes or arguments, beware the will be stored in plain text.
```

Reply:
We could actually scrub these. When storing the input, we could check the action, see if the value corresponds to a sensitive argument or sensitive attribute, and if so replace it with `**redacted**` in the changes? We would also need to traverse any unions/embeds given and scrub those too.

Reply:
I like that approach. Makes dumping the input more difficult. @chrishop let me know if you want to do that or I can put it in another issue for later.

Reply:
From what @zachdaniel is saying it sounds like this needs to apply to `:changes` as well as `:inputs`, so I'm up for doing it later, but I think it should happen outside of this PR.

Reply:
If we don't scrub the values in this PR(which is fine) then we should mark the whole attribute as sensitive then :)

Reply:
made a new issue for it here: https://github.com/ash-project/ash_paper_trail/issues/30
will do a PR off the back of it.